src/modules/m_bansync.cpp

```diff
@@ -0,0 +1,42 @@
+#include "inspircd.h"
+
+/* $ModDesc: Implements extban +b J: - check bans from other channels */
```

Comment:
If you want to get this into -extras then you'll need ModAuthor/ModAuthorMail here too.